Frontend/ui-customer/src/Helpers/Utils.js

```diff
@@ -62,8 +62,19 @@ export const getCurrentDate = function (string) {
   return today;
 }
 
+export const getCurrentDatev2 = function (string) {
+  let today = new Date();
+  const dd = String(today.getDate()).padStart(2, "0");
+  const mm = String(today.getMonth() + 1).padStart(2, "0"); //January is 0!
+  const yyyy = today.getFullYear();
+
+  today = yyyy + "-" + mm + "-" + dd;
+  return today;
+}
+
 
 export const compareDate = function (date1, date2) {
+  console.log(date1, date2);
```

Comment:
Add proper log with identifier  or remove this log

Reply:
done